components/ome-xml/pom.xml

```diff
@@ -46,6 +46,12 @@
     </dependency>
 
     <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
With https://github.com/commit/f2892ff0b9261bfe22515e3ef1d791f86b4fb0bf, I would think this and the corresponding ant dependency addition can be dropped.